addons/mail/static/src/core/web/discuss_patch.xml

```diff
@@ -4,6 +4,9 @@
         <xpath expr="//*[@t-ref='mobileTopbar']" position="before">
             <DiscussSidebar t-if="!ui.isSmall"/>
         </xpath>
+        <xpath expr="//*[@t-ref='root']" position="before">
+            <ControlPanel t-if="!ui.isSmall"/>
```

Comment:
I'm not sure if manually instantiating it is the right way, but if it works I guess that's fine. I wonder how other views do it.

Reply:
There are multiple examples of components using this trick too in the codebase. It should normally be done using the `<Layout>` component and the `display`props but if we surround `<Discuss>`with it, it breaks our CSS. I did not investigate deeper than that.

Reply:
I'm less confident in this change, now in stable I would suggest not messing up with the control panel/breadcrumbs.

And see what we can do in master.